README.md

```diff
@@ -1,2 +1,35 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
+- [Node Version Manager](https://github.com/nvm-sh/nvm)
+- [Homebrew](https://formulae.brew.sh/formula/node)
+- [expressJS](https://expressjs.com/)
+- [NunJucks](https://mozilla.github.io/nunjucks)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README. 
+2. git clone strike off objections web under your "repositories/SingleService" directory you will also require [Strike Off Objections API](https://github.com/companieshouse/strike-off-objections-api)
```

Comment:
neither steps 2 or 3 are neccessary.
Tilt will clone the repo down when you enable development mode.
And you do not need to run make

Reply:
Removed those steps